src/cfddlc_transactions.cpp

```diff
@@ -261,9 +261,13 @@ bool DlcManager::VerifyCetAdaptorSignatures(
   bool all_valid = true;
 
   for (size_t i = 0; i < nb && all_valid; i++) {
+    std::vector<SchnorrPubkey> r_values;
```

Comment:
Let's check the array size.
(It was previously done by DlcManager::ComputeAdaptorPoint().)
```suggestion
    if (oracle_r_values.size() != msgs[i].size()) {
      throw CfdException(CfdError::kCfdIllegalArgumentError,
                       "Number of r values and messages must match.");
    }
    std::vector<SchnorrPubkey> r_values;
```

Reply:
Fixed with https://github.com/p2pderivatives/cfd-dlc/pull/34/files#diff-aade2bd579c5db15761851c4fe892c762a8b5086dcb0371406bab662a739d405R220